src/AzureStack/FabricAdmin/Fabric.Admin/generate.cmd

```diff
@@ -4,4 +4,4 @@
 ::
 
 @echo off
-call %~dp0..\..\..\..\tools\generate.cmd azsadmin/resource-manager/fabric latest Azure current azure-rest-api-specs %CD%
+call %~dp0..\..\..\..\tools\generate.cmd azsadmin/resource-manager/fabric latest deathly809 current azure-rest-api-specs %CD%
```

Comment:
do you have the update meta data txt file that get's generated as part of this generated code?
Also we do not allow you to generate code in public repo from your fork.
Generate from main REST API spec fork.

Reply:
@shahabhijeet What metadata file?  I have never included any metadata files in my PRs and it is not mentioned anywhere in the readme.md.  If there is something that needs to be done with this file then it should be included in the PR process instructions.  Please update the instructions.